src/main/java/network/brightspots/rcv/StreamingCVRReader.java

```diff
@@ -50,6 +50,8 @@
 
 class StreamingCVRReader {
 
+  // this indicates a missing precinct Id in output files
+  private final String MISSING_PRECINCT_ID = "missing_precinct_id";
```

Comment:
This can be a local variable. Same with `tabulationLogPath`.

Reply:
I prefer to pull something like this out into a constant, since we might end up referencing it elsewhere.

Reply:
Any constant can be converted to a local variable.  It's here as a more central location self-documenting location.